Wikidata.js

```diff
@@ -200,12 +202,15 @@ function scrape(doc, url) {
 							if (!value && resource) {
 								//Z.debug("Internal look up resource: " + resource);
 								value = ZU.xpathText(xml, '//rdf:Description[@rdf:about="'+resource+'"]/rdfs:label[contains(@xml:lang, "en")][1]', namespaces)
-									|| ZU.xpathText(xml, '//rdf:Description[@rdf:about="'+resource+'"]/rdfs:label[1]', namespaces);
+									|| ZU.xpathText(xml, '//rdf:Description[ps:P50[rdf:resource="'+resource+'"]]/pq:P1545[1]', namespaces);
+									
 								//Z.debug(value);
 							}
 							if (zprop=="creator") {
+								var seriesOrdinal = ZU.xpathText(xml, '//rdf:Description[ps:P50[@rdf:resource="'+resource+'"]]/pq:P1545[1]', namespaces)
+									|| ZU.xpathText(xml, '//rdf:Description[ps:P2093[text()="'+value+'"]]/pq:P1545[1]', namespaces);
```

Comment:
What happens if the "series ordinal" qualifier is missing? P.S. I have not seen an item where it is missing, but guessing that is possible?

I'd imagine in this case the best behavior would be just to preserve the order provided by wikidata (i.e. the old behavior).

Reply:
That is a good question! In theory: If there is no series ordinal qualifier, then this variable should become None or False and the ordering some lines ago possibly will just do nothing, i.e. the initial order stays the same. However, it might be a good idea to test this in practice... (we can make our example with the sandbox item in wikidata)

Reply:
Tests with https://www.wikidata.org/wiki/Q4115189 seem okay, but I wait until my newest change update in the RDF serialization...

Reply:
Seems to work also fine...